css/style.css

```diff
@@ -15,5 +15,113 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+ h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 2rem 5rem;
+}
+
+.header,
+.content,
+.footer {
+  padding: 0 1.5rem;
+  max-width: 1200px;
+  margin: 0 auto;
+  display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 2rem 5rem;
+}
+
+/*main*/
+main {
+  margin-bottom: 200px;
+}
+
+/*Navigation*/
+
+.nav1 {
+  display: flex;
+  justify-content: center;
+align-items: center;
+flex-wrap: wrap;
+  list-style: none;
+}
+.nav2 {
+  padding: 0.5em 0;
+  display: flex;
+}
+.nav3 {
+  color: var(--grey-dark);
+  font-weight: 600;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin: 8px;
+}
+.nav3:hover {
+  background: #c05326;
+}
+
+/*wifi*/
+.main1 {
+    background-image: url(/img/first-background.jpg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 12rem 0;
+    color: var(--white);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: .5rem;
+}
+.main2 {
+    margin: 2em;
+  text-align: center;
+  flex-direction: column;
+  display: flex
+ 
+}
+.main3 {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 2em;
+  text-align: center;
+  flex-wrap: wrap;
+  
+}
 
+/*footer*/
+.footer1 {
+  
+  
+}
 
+.footer2 {
+  border-top: 1px solid var(--grey-light);
+  padding-top: 2rem;
+  padding-left: 550px;
+  padding-bottom: 2rem;
+  
+}
+.footer3 {
+  padding-bottom: 1rem;
+  
+  text-align: center;
+  font-size: 0.5rem;
+  display: flex;
+}
+button {
+    background-color: #c05326;
+
+}
+.hi .hi2 .hi3 {
+    padding: 0 15px;
+}
```

Comment:
try to have logical names for classes/variables; the classes/variable names should indicate what they represent so as not to rely on future developers having all the context to understand them